tests/main.nim

```diff
@@ -95,6 +95,30 @@ suite "redis tests":
     discard r.pfadd("redisTest:pfcount2", @["bar"])
     check r.pfcount(@["redisTest:pfcount1", "redisTest:pfcount2"]) == 2
 
+  test "sorted sets":
+    const expected = 1
+
+    discard r.zadd("redisTest:myzset", 1, "one")
+    discard r.zadd("redisTest:myzset", 2, "two")
+    discard r.zadd("redisTest:myzset", 3, "three")
+
+    assert r.zcard("redisTest:myzset") == 3
+
+    assert r.zcount("redisTest:myzset", "-inf", "+inf") == 3
+    assert r.zcount("redisTest:myzset", "(1", "3") == 2
```

Comment:
Is this `(` intentional?

Reply:
yeah, its a special marker, `(1` means `after 1` (sorry my English is pretty bad)